dcspy/qt_gui.py

```diff
@@ -608,7 +614,12 @@ def _checked_iface_rb_for_identifier(self, key_name: str) -> None:
             widget_iface = self.input_reqs[self.current_plane][key_name].widget_iface
             self.le_custom.setText('')
             if widget_iface == 'rb_custom':
-                self.le_custom.setText(self.input_reqs[self.current_plane][key_name].request.split('CUSTOM ')[1])
+                custom_request = self.input_reqs[self.current_plane][key_name].request.split('CUSTOM ')[1]
+                # if there is a request value then use this, otherwise if this is a push_button then default to BUTTON
```

Comment:
This is temporary and intended to be replaced with an actual radio button in later commits.

Reply:
Her the same we need another if for BUTTON.